syncmlcommon/SyncMLCommon.h

```diff
@@ -90,4 +92,8 @@ const QString SYNCML12                = "SyncML12";
 
 const QString PROF_HTTP_XHEADERS      = "http_xheaders";
 
+
+Q_DECLARE_LOGGING_CATEGORY(lcSyncMLPlugin)
```

Comment:
should this be lcButeoSyncMLPlugin or?  I guess it doesn't matter

Reply:
lcSyncMLPlugin is probably sufficient since it's only used within this repo.